client/server/isomorphic-routing/index.js

```diff
@@ -1,13 +1,20 @@
+import debugFactory from 'debug';
 import { isEmpty } from 'lodash';
 import { stringify } from 'qs';
 import { setSectionMiddleware } from 'calypso/controller';
 import { serverRender, setShouldServerSideRender } from 'calypso/server/render';
 import { setRoute } from 'calypso/state/route/actions';
 
+const debug = debugFactory( 'calypso:pages' );
```

Comment:
`calypso_pages` can be a bit ambiguous here. We have a `pages` section and one could confuse this debug flag with that.

Reply:
That's actually what I was trying to accomplish -- though I agree it's a bit weird! I wanted logs in this file to show up when debugging pages, because both files are linked (e.g. the router is mostly called from the pages module when the pages module sets up sections)

Reply:
I can change it to something else if you think that's a better idea :)